vowpalwabbit/example.cc

```diff
@@ -308,6 +308,97 @@ void free_flatten_example(flat_example* fec)
   }
 }
 
+std::string features_to_string(const example& ec)
```

Comment:
are all these to_string functions only for debugging purposes? (and the ones below)

Reply:
Yes.